tests/SchemaTest.php

```diff
@@ -525,4 +525,37 @@ public function testNotConnectionPDO(): void
 
         $schema->refreshTableSchema('customer');
     }
+
+    public function testDomainType(): void
+    {
+        $db = $this->getConnection();
+
+        $command = $db->createCommand();
+        $schema = $db->getSchema();
+
+        if ($schema->getTableSchema('test_domain_type') !== null) {
+            $command->dropTable('test_domain_type')->execute();
+        }
+
+        $command->setSql('DROP DOMAIN IF EXISTS sex_char')->execute();
+        $command->setSql(
+            'CREATE DOMAIN sex_char AS "char" NOT NULL DEFAULT \'x\' CHECK (VALUE in (\'m\', \'f\', \'x\'))'
+        )->execute();
+
+        $command->createTable('test_domain_type', ['id' => 'pk', 'sex' => 'sex_char'])->execute();
```

Comment:
```suggestion
        $command->setSql('DROP DOMAIN IF EXISTS sex_char')->execute();
        $command->setSql(
            'CREATE DOMAIN sex_char AS "char" NOT NULL DEFAULT \'x\' CHECK (VALUE in (\'m\', \'f\', \'x\'))'
        )->execute();
        
        if ($schema->getTableSchema('test_domain_type') !== null) {
            $command->dropTable('test_domain_type')->execute();
        }
        $command->createTable('test_domain_type', ['id' => 'pk', 'sex' => 'sex_char'])->execute();
```

Suggestion only for better reading of test.